sirius-pipeline/bmv2/sirius_outbound.p4

```diff
@@ -62,7 +62,7 @@ control outbound(inout headers_t hdr,
         key = {
             /* Flow for express route */
             meta.encap_data.dest_vnet_vni : exact @name("meta.encap_data.dest_vnet_vni:dest_vni");
-            hdr.ipv4.dst_addr : exact @name("hdr.ipv4.dst_addr:dip");
+            meta.dst_ip_addr : exact @name("meta.dst_ip_addr:dip");
```

Comment:
There needs to be a flag to not overlap v4 and v6 prefixes that have identical 4 LSB.

Reply:
@marian-pritsak How do I achieve this in P4?  How do I define this flag?  Do you have some example?  Thanks!

Reply:
Another field bit<1> in metadata, used as match key as well

Reply:
@marian-pritsak Thanks.  I have made the requested changes.  I will update the PR by tonight.

Reply:
@marian-pritsak I have pushed the requested changes to the PR.  Please review.  Thanks!